docs/visual-basic/programming-guide/language-features/linq/introduction-to-linq.md

```diff
@@ -44,84 +44,158 @@ Language-Integrated Query (LINQ) adds query capabilities to Visual Basic and pro
   
 -   [How To and Walkthrough Topics](#HowToAndWalkthroughTopics)  
   
-##  <a name="RunningtheExamples"></a> Running the Examples  
+##  <a name="RunningtheExamples"></a> Running the examples  
```

Comment:
not related to your change and I'm happy to open a separate PR for this if you'd like: I find these anchors so ugly. I understand using them with complicated anchors but this would be running-the-examples. 

We could just remove the mini TOC from the top.

Reply:
I've removed the anchors, @mariaw.